host/src/gap.rs

```diff
@@ -87,8 +94,11 @@ impl<'a> GapConfig<'a> {
     ///
     /// This configuration will use the `GENERIC_UNKNOWN` appearance.
     pub fn default(name: &'a str) -> Self {
+        static NAME_BYTES: StaticCell<Vec<u8, DEVICE_NAME_MAX_LENGTH>> = StaticCell::new();
+        let name_bytes = NAME_BYTES.init(Vec::new());
+        fill_vec(name_bytes, name.as_bytes());
```

Comment:
This will panic if you call `default` more than once.

Reply:
Oof yes you're right. I'm planning to replace this implementation anyway as it should be possible to reference the string without having to copy it I think.